Source/WebCore/platform/mediarecorder/MediaRecorderPrivateEncoder.cpp

```diff
@@ -905,11 +905,12 @@ void MediaRecorderPrivateEncoder::fetchData(CompletionHandler<void(RefPtr<Fragme
         return flushPendingData(currentTime)->whenSettled(queueSingleton(), [protectedThis, this, completionHandler = WTFMove(completionHandler), currentTime]() mutable {
             assertIsCurrent(queueSingleton());
             RefPtr data = takeData();
```

Comment:
Drive-by: since it's assumed data is non-null, would it be cleaner to assign to a Ref?